cmd/zdb/zdb.c

```diff
@@ -8186,6 +8255,7 @@ main(int argc, char **argv)
 		case 'm':
 		case 'M':
 		case 'O':
+		case 'r':
```

Comment:
Oh could we have 'r' added to usage as well? I spent forever wondering why `zdb -h` wasn't showing the new option :)